_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.spec.ts

```diff
@@ -187,6 +196,9 @@ describe('smart-shopping-campaign-creation.vue - Campaign edition - Unhandled fi
   let wrapper;
   const mockRoute = {
     name: 'campaign-edition',
+    params: {
+      type: 'SMART_SHOPPING_CAMPAIGN',
+    },
```

Comment:
There is a suggestion in https://github.com/PrestaShopCorp/psxmarketingwithgoogle/pull/1058 that would prevent the route parameters to be added